packages/core/src/Button/Button.stories.tsx

```diff
@@ -141,52 +192,34 @@ export const Icons: StoryObj<HvButtonProps> = {
     <>
       <div>
         <HvButton icon aria-label="Play" variant="primaryGhost">
-          <Play iconSize="M" />
+          <Play />
```

Comment:
I just noticed... these were the only samples testing that `HvButton` "behaves well" with larger content. The remaining samples are all "sm" icons and/or normal-sized text.
Maybe there's value in keeping something like this, what do you think? 🤔

Reply:
I remove this because DS specs always use icons with 32px but I'll add the icon sizes back to test how the button behaves

Reply:
✅